src/app/organizer/ItemTable.tsx

```diff
@@ -309,6 +309,7 @@ function ItemTable({
             if (e.shiftKey) {
               if ((e.target as Element).hasAttribute('data-perk-name')) {
                 const filter = column.filter!(
+                  // eslint-disable-next-line @typescript-eslint/no-unnecessary-type-assertion
                   (e.target as Element).getAttribute('data-perk-name')!,
```

Comment:
It's asking you to remove the `!` from the end of this line because filter apparently can take `undefined/null` now.

Reply:
oh, did not even see the !. it looked `as` related